.eslintrc.json

```diff
@@ -231,7 +245,10 @@
       "rules": {
         // type declarations for other libraries don't get to choose their
         // export structure
-        "import/prefer-default-export": ["off"]
+        "import/prefer-default-export": "off",
+        // we often need to declare unused variables when extending generic
+        // types
+        "@typescript-eslint/no-unused-vars": "off"
```

Comment:
Actually, is this the rule that keeps us from accidentally forgetting to await promises?  I think this is probably useful to keep around if we don't mind having a few localized eslint-disables running around.

Reply:
Answering my own question: I think that's `no-misused-promises`; this is about creating variable bindings that are not used for anything else.

Reply:
Yeah that's right. Also, if you expand the context a bit, this rule configuration is specifically scoped to anything under `types/`, so won't apply to general code (in general, I think this is a good rule!)